Automattic/jetpack
# Backup: replace ConnectButton with ConnectScreen

#### Changes proposed in this Pull Request:

* In the Backup plugin, update current `<ConnectButton>` component with `<ConnectScreen>`.

#### Jetpack product discussion

* Internal: pbuNQi-1w3-p2
* Internal, task: 1199658893173824-as-1200603127282613/f
> The ConnectScreen will display when not connected...Manage your connection will only appear when actually connected

#### Does this pull request change what data or activity we track or use?

* No.

#### Testing instructions:

* Disconnect and deactivate Jetpack plugin.
* After pulling down these changes, activate the Jetpack Backup plugin and navigate to `/wp-admin/admin.php?page=jetpack-backup`
* You should see the ConnectScreen component active (`jp-connect-screen`), click the `Set up Jetpack` button to connect site and user.
